notebooks/dynamicrafter-animating-images/dynamicrafter-animating-images.ipynb

```diff
@@ -73,33 +73,18 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    gc.collect()
</pre><p>please add ; after gc.collect() for avoid reprorting confusing logs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
done